src/common/i18n/translations/en.ts

```diff
@@ -106,5 +106,270 @@ export const en: Translations = {
+      primaryActionText: "Restart app"
+    },
+    wrongFormatCountryCode: {
+      title: "Wrong format",
+      body: "Enter a valid country code.",
+      primaryActionText: "OK"
+    },
+    wrongFormatCountryCodePhoneNumber: {
+      title: "Wrong format",
+      body: "Enter a valid country code and contact number.",
+      primaryActionText: "OK"
+    },
+    wrongFormatQRScanAgain: {
+      title: "Wrong format",
+      body: "Scan QR code again or get a new QR code from your in-charge.",
+      primaryActionText: "OK"
+    },
+    wrongFormatValidCode: {
+      title: "Wrong format",
+      body: "Enter valid code details.",
+      primaryActionText: "OK"
+    },
+    wrongFormatDeviceCode: {
+      title: "Wrong format",
+      body: "Scan device code again or get a new device from your in-charge.",
+      primaryActionText: "OK"
+    },
+    wrongFormatID: {
+      title: "Wrong format",
+      body: "Enter or scan a valid ID number.",
+      primaryActionText: "OK"
+    },
+    wrongFormatQRGetNew: {
+      title: "Wrong format",
+      body: "Get a new QR code from your in-charge.",
+      primaryActionText: "OK"
+    },
+    alreadyUsedCode: {
+      title: "Already Used",
+      body: "Enter unique code details",
+      primaryActionText: "OK"
+    },
+    alreadyUsedItem: {
+      title: "Already Used",
+      body: "Scan another item that is not tagged to any ID number",
+      primaryActionText: "OK"
+    },
+    wrongFormatCode: {
+      title: "Wrong format",
+      body: "Enter or scan valid code details",
+      primaryActionText: "OK"
+    },
+    incompleteEntryCode: {
+      title: "Incomplete entry",
+      body: "Enter or scan code details",
+      primaryActionText: "OK"
+    },
+    wrongFormatNotValidDeviceCode: {
+      title: "Wrong format",
+      body: "Scan valid device code",
+      primaryActionText: "OK"
+    },
+    incompleteEntryScanDeviceCode: {
+      title: "Incomplete entry",
+      body: "Scan device code",
+      primaryActionText: "OK"
+    },
+    wrongFormatContactNumber: {
+      title: "Wrong format",
+      body: "Enter valid contact number",
+      primaryActionText: "OK"
+    },
+    insufficientQuota: {
+      title: "Error",
+      body: "Insufficient quota",
+      primaryActionText: "OK"
+    },
+    invalidQuantity: {
+      title: "Error",
+      body: "Invalid quantity",
+      primaryActionText: "OK"
+    },
+    categoryDoesNotExist: {
+      title: "Error",
+      body: "Category does not exist",
+      primaryActionText: "OK"
+    },
+    systemErrorServerIssues: {
+      title: "System error",
+      body:
+        "We are currently facing server issues. Try again later or contact your in-charge if the problem persists",
+      primaryActionText: "OK"
+    },
+    discardTransaction: {
+      title: "Discard transaction?",
+      body: "This will clear all scanned items",
+      primaryActionText: "Cancel",
+      secondaryActionText: "Discard"
```

Comment:
Are these accidentally inverted? Cuz when i click on "Cancel" it discards my transactions, and when I click "Discard" it cancels my "discard transaction" action

Reply:
Same for chinese

Reply:
Good catch, have switched both English and Chinese translation `primaryActionText` and `secondaryActionText`!